wpilibc/shared/src/Commands/Scheduler.cpp

```diff
@@ -47,7 +47,7 @@ void Scheduler::AddCommand(Command* command) {
 
 void Scheduler::AddButton(ButtonScheduler* button) {
   std::lock_guard<priority_mutex> sync(m_buttonsLock);
-  m_buttons.push_back(button);
+  m_buttons.emplace_back(button);
```

Comment:
I thought `emplace_back` called a constructor, forwarding arguments to it. Is this what's happening here?